crates/providers/blobstore-fs/src/lib.rs

```diff
@@ -241,7 +242,7 @@ impl FsProvider {
 
         let cr = receiver.receive_chunk(chunk).await
             .with_context(|| format!(
-                "sending chunk error: Container({container_id}) Object({object_id}) to Actor({actor_id})",
+                "sending chunk error: Container({container_id}) Object({object_id}) to Actor({source_id})",
```

Comment:
For the blobstore provider, where the provider is the think linked _to_ a component, I think we should be sending chunks to the `target` here on the linkdef.

Since the blobstore provider is one that links to components and has components link to it (with interface linkdefs) I think this is a good example for us to find out how to properly branch the code based on that. Could be as simple as matching the source_id to see if it's the provider ID or if it's something else.

Reply:
I thought it would still be the source -- because this is used via the look up the link that *came in*, right?

The link that is saved is the one that is made when the actor first connects, and the source=actor, target=provider.

Reply:
Well, it's tricky, but for this situation the actor will need to call the provider and vice versa, so there will need to be a link with the provider as the target and the provider as the source

Reply:
Oh wait I'm misunderstanding, you mean that in this case the actor calls the provider, and so the provider is just sending chunks back to the original caller. That makes sense!

It does bring up a question on if we want to validate that there is in fact a link from the provider to the actor when calling `wasmcloud:blobstore/chunk-receiver.receive-chunk`, in a pure world we would require this and so we could either
1. Enforce it at the host level (deny the invocation since the component didn't have a link to it)
2. Enforce it at the provider level (subject to implementation details

I think this is a little bit weird and I need to think about it a little bit more, but it doesn't affect this PR merging

Reply:
> Oh wait I'm misunderstanding, you mean that in this case the actor calls the provider, and so the provider is just sending chunks back to the original caller. That makes sense!

Yup!

Yeah the links no longer being bi-directional actually does bring up a rather interesting issue. We basically need a `put_link` on Actors as well... Should hosts be intermediating links that are made to actors, or should actors do that themselves?

Feels like the distinction between actor and provider continues to dwindle!